code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -98,8 +98,7 @@
 
 /obj/effect/overlay/temp/at_shield/New()
 	..()
-	spawn()
-		orbit(target, 0, FALSE, 0, 0, FALSE, TRUE)
+	orbit(target, 0, FALSE, 0, 0, FALSE, TRUE)
```

Comment:
unless I'm mistaken, because orbit sleeps, this would block whatever created this until the orbit ended.

should keep as is or use a `set waitfor = 0` proc